docs/streams/upgrade-guide.html

```diff
@@ -141,6 +141,91 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
+        <li><strong>Sticky Task Assignor:</strong> A basic task assignment strategy that minimizes task movement
+            during rebalances is included.</li>
+        <li><strong>Interactive Query Support:</strong> IQ operations are compatible with the new streams protocol.</li>
+        <li><strong>New Admin RPC:</strong> The <code>StreamsGroupDescribe</code> RPC provides streams-specific metadata
+            separate from consumer group information, with corresponding access via the <code>Admin</code> client.</li>
+        <li><strong>CLI Integration:</strong> You can list, describe, and delete streams groups via the <code>kafka-streams-groups.sh</code> script.</li>
+    </ul>
+
+    <p><strong>What's Not Included in Early Access</strong></p>
+
+    <ul>
+        <li><strong>Static Membership:</strong> Setting a client `instance.id` will be rejected.</li>
+        <li><strong>Topology Updates:</strong> If a topology is changed significantly (e.g., by adding new source topics
+            or changing the number of sub-topologies), a new streams group must be created.</li>
+        <li><strong>High Availability Assignor:</strong> Only the sticky assignor is supported.</li>
+        <li><strong>Regular Expressions:</strong> Pattern-based topic subscription is not supported.</li>
+        <li><strong>Reset Operations:</strong> CLI offset reset operations are not supported.</li>
+        <li><strong>Protocol Migration:</strong> No automated migration is available between the classic and new streams protocols.</li>
+    </ul>
+
+    <p><strong>Why Use the Streams Rebalance Protocol?</strong></p>
+
+    <ul>
+        <li>
+            <strong>Broker-Driven Coordination:</strong>
+            Centralizes task assignment logic on brokers instead of the client. This provides consistent,
+            authoritative task assignment decisions from a single coordination point and reduces the potential for
+            split-brain scenarios.
+        </li>
+        <li>
+            <strong>Faster, More Stable Rebalances:</strong>
+            Reduces rebalance duration and impact by removing the global synchronization point. This minimizes
+            application downtime during membership changes or failures.
+        </li>
+        <li>
+            <strong>Better Observability:</strong>
+            Provides dedicated metrics and admin interfaces that separate streams from consumer groups, leading to
+            clearer troubleshooting with broker-side observability.
+        </li>
+    </ul>
+
+    <p>
+        To enable the protocol, set <code>unstable.feature.versions.enable=true</code> for controllers and brokers, and
+        set <code>unstable.api.versions.enable=true</code> on the brokers as well. In your Kafka Streams application
+        configuration, set <code>group.protocol=streams</code>.
+    </p>
+
+    <p>
+        Migration between the classic consumer group protocol and the Streams Rebalance Protocol is not supported in
```

Comment:
It should be possible to delete a group, to free up the `group.id` to switch between protocols, right? Should we add a sentence about it somewhere in this paragraph?